### PR TITLE
 Add release note for IntelliJ 3.20.0

### PR DESCRIPTION
**Pull request by Flanker32**
@wezhang Please provide details for the HDInsight part.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,6 +31,19 @@ All notable changes to "Azure Toolkit for IntelliJ IDEA" will be documented in t
   - [3.0.7](#307)
   - [3.0.6](#306)
 
+## [3.20.0]
```
Comment (andxu) [inline CHANGELOG.md]: remove the bracket

**Comment by t-rufang** [comment]
Thanks @Flanker32 ! Could you help to add the following items to release notes?

#### Added
 - Support Failure Task Local Reproduce for Spark 2.3 on Cosmos
 - Support mock file system in Spark local console
 - Support ADLS Gen2 storage type to submit job to HDInsight cluster
 - Introduce extended properties field when provision a Spark on Cosmos cluster or submit a Spark on Cosmos Serverless job
#### Changed
 - Change icons for HDInsight cluster and related configuration
#### Fixed
 - #2888, #2894, #2921 Fix Spark on Cosmos Serverless job run failed related issues
 - #2912 Check invalid access key for submitting with ADLS Gen2 account
 - #2844 Refine WebHDFS and ADLS input path hints
 - #2848 Reset background color for not empty ADLS path input
 - #2749, #2936 Fix Spark run configuration cast issues and classified exception message factory NPE issues

File: PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -20,26 +20,40 @@
...
-        <li><a href="https://github.com/Microsoft/azure-tools-for-java/issues/2810" rel="nofollow">#2810</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2760" rel="nofollow">#2760</a> Spark Run Configuration issues when created from context menu.</li>
-      </ul>
-      <p>You may get the full change log <a href="https://github.com/Microsoft/azure-tools-for-java/blob/develop/CHANGELOG.md">here</a></p>
+    <h3>[3.20.0]</h3>
+    <h4>Added</h4>
+    <ul>
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml]: Shall we have a little program in CI generate this html file automatically?

Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml]: Yes, may be we could generate this html from CHANGELOG.md, but we still need to write the chagelog markdown

Comment (andxu) [reply PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml]: I mean from md to html

**Pull request merged**